starter-code/css/style.css

```diff
@@ -0,0 +1,116 @@
+
+
+/* ------------ Typography ------------- */
+
+.logo-nav h2{
+    font-weight: bolder;
+}
+.text-img h3{
+    color: white;
+    font-size: 32px;
+    text-transform: uppercase;
+    width: 80%;
+    margin-left: 10%;
+}
+.text-img p{
+    color: white;
+    font-size: 16px;
+    width: 80%;
+    margin-left: 10%;
+}
+.icons-info{
+    text-align: center;
+}
+.icons-ctn h3{
+    font-size: 24px;
+    font-weight: bolder;
+    color: white;
+    margin-bottom: 10px;
+}
+.icons-ctn p{
+    font-size: 14px;
+    color: white;
+}
+
+
+
+
+
+
+/* ------------ Layout -------------- */
+
+.nav-bar{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 100px;
+}
+.icons-info{
+    display: flex;
+    justify-content: space-around;
+}
+
+
+
+
+
+
+/* ------------ Components ----------- */
+
+.nav-bar{
+    margin-right: 20px;
+}
+.nav-menu ul{
+    margin: 10px;
+}
+.nav-menu ul  li{
+    list-style-type: none;
+    display: inline;
+    padding: 10px;
+    text-align: center;
+    font-size: 14px;
+}
+
+.overlay{
+    background-image: url('../img/overlay.png');
+    position: absolute;
```

Comment:
Position: absolute is making this div looking overlayed. It is hiding things behind.